File: test/fixtures/node-superagent/commented_json_with_session.js
import request from 'superagent';

request
  .post('http://localhost:28139')
  .set('Accept', 'application/json')
  .type('json')
  //  .send('{   }')
  .send({})
  .then(res => {
     console.log(res.body);
  });

File: test/fixtures/ruby-httparty/commented_json_with_session.rb
require 'httparty'
require 'json'

url = 'http://localhost:28139'
headers = {
  'Content-Type': 'application/json',
  'Accept': 'application/json',
}
# The object won't be serialized exactly like this
# body = '{   }'
body = {}.to_json
res = HTTParty.post(url, headers: headers, body: body)
